src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private static final int randomMax = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(randomMax));
+    }
```

Comment:
Привіт, якщо якась змінна private static final на рівні класу і це примітив, або незмінний (immutable) об'єкт (наприклад String) - то це константа і її назва має бути капсом із роділенням підкреслюванням між словами (CONSTANT_NAME)
І зазвичай константи пишуть першими, перед іншими змінними

якщо це не private static final, або якщо це змінна методу, то її назва має бути в camel case (variableName)

```suggestion
    private static final int RANDOM_MAX = 100;
    
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();
    
    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(RANDOM_MAX));
    }
```